src/crypto-api.ts

```diff
@@ -840,6 +806,11 @@ export interface CreateSecretStorageOpts {
      */
     setupNewSecretStorage?: boolean;
 
+    /**
+     * Create a dehydrated device if no dehydrated device is already present.
+     */
+    initialiseDeviceDehydration?: boolean;
+
```

Comment:
this doesn't seem to be implemented?

Reply:
Right, I tried to add it, changed my mind, and missed removing this bit.